gradebook/grade/models.py

```diff
@@ -0,0 +1,112 @@
+from django.db import models
+
+class User(auth.models.AbstractUser):
+    id = models.IntegerField()
```

Comment:
You don't have to define `id` field, as it taken care of by Django.

Besides, to make it primary key, you need to pass to it `primary_key=True`.